apps/web/src/server/views/validation/review-appeal.njk

```diff
@@ -7,6 +7,16 @@
 {% from "app/components/document-list.component.njk" import documentList %}
 
 {% set isAppealIncomplete = appeal.AppealStatus === 'incomplete' %}
+{% from "_includes/macros/make-appeal-site-html.njk" import makeAppealSiteHtml %}
+
+{# Macro use to generate a list of documents #}
+{% macro supportingDocumentsList(supportingDocuments)  %}
```

Comment:
You can use the existing  documentList(supportingDocuments) for this

Reply:
I forgot to remove this after merging. The documentList macro didn't exist when I originally wrote this.